src/classes/OPP_StageMappingCMT_TEST.cls

```diff
@@ -0,0 +1,91 @@
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING,
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+/**
+* @author Salesforce.org
+* @date 2019
+* @group Opportunity
+* @group-content ../../ApexDocContent/Opportunity.htm
+* @description Test Coverage for OPP_StageMappingCMT class
+*/
+@isTest(isParallel=true)
+private with sharing class OPP_StageMappingCMT_TEST {
+
+    /*********************************************************************************************************
+    * @description Tests the createFinalizedOppStageToStateRecord Method and creates the finalized record. 
+    * @return void
+    **********************************************************************************************************/
+    @isTest
+    private static void testFinalizedOppStageToStateRecordCreation() {
+        String closedWonStage = UTIL_UnitTestData_TEST.getClosedWonStage();
+        npe01__Contacts_And_Orgs_Settings__c configRecord = new npe01__Contacts_And_Orgs_Settings__c();
+        configRecord.Payments_Auto_Close_Stage_Name__c = closedWonStage;
+
+        npe01__Contacts_And_Orgs_Settings__c contactAndOrgSetting = UTIL_CustomSettingsFacade.getContactsSettingsForTests(configRecord);
+        UTIL_CustomSettingsFacade.getContactsSettings().Payments_Auto_Close_Stage_Name__c = closedWonStage;
+        
+        Test.startTest();
+        // We cannot test the result of this call since the response to this call is async. 
+        // The best we can do right now is just execute the code. 
+        List<Metadata.CustomMetadata> metadataToInsert = new List<Metadata.CustomMetadata>();
+        Map<String, Object> params = new Map<String, Object>();
+
+        metadataToInsert = Opp_StageMappingCMT.createFinalizedOppStageToStateRecord(metadataToInsert);
+
+        Opp_StageMappingCMT.deployOppStageToStateMetadata(metadataToInsert, params);
+        Test.stopTest();
+
+
+    }
+
+    /*********************************************************************************************************
+    * @description Tests the createPledgedOppStageToStateRecord Method and creates the pledged record. 
+    * verifies that it returns the appropriate values
+    * @return void
+    **********************************************************************************************************/
+    @isTest
+    private static void testPledgedOppStageToStateRecordCreation() {
+        String closedWonStage = UTIL_UnitTestData_TEST.getClosedWonStage();
+        npe01__Contacts_And_Orgs_Settings__c configRecord = new npe01__Contacts_And_Orgs_Settings__c();
```

Comment:
Same change here to reduce the 3 lines into one.

Reply:
Updated